src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  cursor: pointer;
+  margin-right: auto;
+  margin-left: 50px;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+}
+
+.nav__list a {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  margin-left: 16px;
+}
+
+.nav__list li a {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav__list li {
+  display: inline-block;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  text-transform: uppercase;
+}
+
+.is-active {
+  margin-left: 0;
+  color: #00acdc;
+  position: relative;
+}
+
+.nav__list li a:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
```

Comment:
no tags for styles and no increasing specificity either

Reply:
The point here is that if I remove the tags from .nav__list, the styles stop working. I used the tags because they let me pass the test, if I just style .nav__list most styles don't apply anymore.

Reply:
> The point here is that if I remove the tags from .nav__list, the styles stop working. I used the tags because they let me pass the test, if I just style .nav__list most styles don't apply anymore.

that is because when you have `.nav__list li` you are styling the `li`, which is the list item. if you remove `li` from the selector you no longer styling the `li` element, you are styling the list (`ul`). give your `li` a class and style it instead of `.nav__list li`